{"metadata":{"role":"collectionGroup","modules":[{"name":"SwiftPy"}],"roleHeading":"API Collection","title":"LocalizedError Implementations"},"identifier":{"interfaceLanguage":"swift","url":"doc:\/\/SwiftPy\/documentation\/SwiftPy\/InterpreterError\/LocalizedError-Implementations"},"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/swiftpy\/interpretererror\/localizederror-implementations"]}],"hierarchy":{"paths":[["doc:\/\/SwiftPy\/documentation\/SwiftPy","doc:\/\/SwiftPy\/documentation\/SwiftPy\/InterpreterError"]]},"schemaVersion":{"minor":3,"major":0,"patch":0},"topicSections":[{"title":"Instance Properties","generated":true,"identifiers":["doc:\/\/SwiftPy\/documentation\/SwiftPy\/InterpreterError\/failureReason","doc:\/\/SwiftPy\/documentation\/SwiftPy\/InterpreterError\/helpAnchor","doc:\/\/SwiftPy\/documentation\/SwiftPy\/InterpreterError\/recoverySuggestion"],"anchor":"Instance-Properties"}],"kind":"article","sections":[],"references":{"doc://SwiftPy/documentation/SwiftPy/InterpreterError":{"kind":"symbol","identifier":"doc:\/\/SwiftPy\/documentation\/SwiftPy\/InterpreterError","type":"topic","url":"\/documentation\/swiftpy\/interpretererror","title":"InterpreterError","abstract":[],"role":"symbol","fragments":[{"text":"enum","kind":"keyword"},{"text":" ","kind":"text"},{"text":"InterpreterError","kind":"identifier"}],"navigatorTitle":[{"text":"InterpreterError","kind":"identifier"}]},"doc://SwiftPy/documentation/SwiftPy/InterpreterError/helpAnchor":{"url":"\/documentation\/swiftpy\/interpretererror\/helpanchor","fragments":[{"text":"var","kind":"keyword"},{"text":" ","kind":"text"},{"text":"helpAnchor","kind":"identifier"},{"text":": ","kind":"text"},{"preciseIdentifier":"s:SS","text":"String","kind":"typeIdentifier"},{"text":"?","kind":"text"}],"type":"topic","title":"helpAnchor","role":"symbol","identifier":"doc:\/\/SwiftPy\/documentation\/SwiftPy\/InterpreterError\/helpAnchor","kind":"symbol","abstract":[]},"doc://SwiftPy/documentation/SwiftPy/InterpreterError/recoverySuggestion":{"kind":"symbol","role":"symbol","title":"recoverySuggestion","abstract":[],"fragments":[{"text":"var","kind":"keyword"},{"text":" ","kind":"text"},{"text":"recoverySuggestion","kind":"identifier"},{"text":": ","kind":"text"},{"text":"String","kind":"typeIdentifier","preciseIdentifier":"s:SS"},{"text":"?","kind":"text"}],"url":"\/documentation\/swiftpy\/interpretererror\/recoverysuggestion","type":"topic","identifier":"doc:\/\/SwiftPy\/documentation\/SwiftPy\/InterpreterError\/recoverySuggestion"},"doc://SwiftPy/documentation/SwiftPy":{"identifier":"doc:\/\/SwiftPy\/documentation\/SwiftPy","title":"SwiftPy","role":"collection","url":"\/documentation\/swiftpy","abstract":[],"kind":"symbol","type":"topic"},"doc://SwiftPy/documentation/SwiftPy/InterpreterError/failureReason":{"abstract":[],"kind":"symbol","identifier":"doc:\/\/SwiftPy\/documentation\/SwiftPy\/InterpreterError\/failureReason","type":"topic","url":"\/documentation\/swiftpy\/interpretererror\/failurereason","title":"failureReason","role":"symbol","fragments":[{"text":"var","kind":"keyword"},{"text":" ","kind":"text"},{"text":"failureReason","kind":"identifier"},{"text":": ","kind":"text"},{"text":"String","kind":"typeIdentifier","preciseIdentifier":"s:SS"},{"text":"?","kind":"text"}]}}}